docs/usage/dapi.md

```diff
@@ -24,5 +24,5 @@ const transporter = new DAPIClient({
       timeout: 20000,
       retries: 5,
 });
-const wallet = new Wallet({transporter:transporter});
+const wallet = new Wallet({transport:transporter});
```

Comment:
You need to create a `DAPIClientTransport` and pass the `DAPIClient` instance there.